Source/Core/DolphinQt2/Config/CheatWarningWidget.h

```diff
@@ -0,0 +1,36 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <QWidget>
+
+#include <string>
+
+class QLabel;
+class QPushButton;
+class QTimer;
+
+class CheatWarningWidget : public QWidget
+{
+  Q_OBJECT
+public:
+  explicit CheatWarningWidget(const std::string& game_id);
+
+signals:
+  void OpenCheatEnableSettings();
+  void CheatEnableToggled();
```

Comment:
What is `CheatEnableToggled`?

Reply:
I don't need it anymore, will just throw it.